JS-Algorithms/challenges.js

```diff
@@ -13,7 +17,22 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  
+  if( seconds < 0 || isNaN(seconds) ){
+    return null;
+  }
+  const SECONDS_PER_DAY = 3600 * 24;
+  const SECONDS_PER_HOUR = 3600;
+  const SECONDS_PER_MINUTE = 60;
+  
+  const secs = seconds % SECONDS_PER_DAY;
```

Comment:
Could this variable name be more specific? If I initially want to name a variable a name or close to a name (seconds to sec) to one that already exists, I need to be more specific.